### PR TITLE
feat(css/parser): comments supporting

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Partial https://github.com/swc-project/swc/issues/5384

**BREAKING CHANGE:**

Yes

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_codegen/tests/fixture/comments/input.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* Leading */
+
+a {
+    background: /* Leading */ red /* Trailing */;
+}
+
```
Comment (yisibl) [inline crates/swc_css_codegen/tests/fixture/comments/input.css:6]: Nice work! 

Can you add some comments in the function and in the at rule?

e.g.

```css
@media (width >/* foo */= 30em) {
 .test {
    width: calc(100%-/* foo */20px);
  }
}
```

Comment (alexander-akait) [reply crates/swc_css_codegen/tests/fixture/comments/input.css:6]: @yisibl Yes, I am still thinkig how better solve it, here just example how we should store them

Comment (kdy1) [reply crates/swc_css_codegen/tests/fixture/comments/input.css:6]: Can you resolve this issue?

Comment (alexander-akait) [reply crates/swc_css_codegen/tests/fixture/comments/input.css:6]: Yes, planned, want to finish some stuff before it

**Comment by arHSM** [comment]
Any ETA on getting this merged?

**Comment by kdy1** [comment]
Closing as stale

**Pull request closed without merging**